content/posts/newsletter-011/index.md

```diff
@@ -269,6 +284,44 @@ Also, check out [a talk about open source and Veloren][veloren-talk]:
 [veloren-docs]: https://docs.veloren.net/veloren_voxygen/index.html
 [veloren-talk]: https://youtube.com/watch?v=aS26sqT09Pw
 
+### [Zero to Game][zerotoga.me]
```

Comment:
should this be in "learning materials" section?

Reply:
Hmm. It may be systematized as learning material, but i'd say it's more of a game project, just with a very detailed devlog.